less/home-page.less

```diff
@@ -45,31 +85,118 @@
     padding: 50px 60px;
     display: flex;
     justify-content: center;
-
+    @media @mobile {
+      
+      width: 100%;
+      padding-top: 20px;
+      padding-bottom: 20px;
+    
+    }
+  
     .walk-text {
       width: 50%;
+      @media @mobile {
+        width: 100%;
+        
+      }
 
       h3 {
         margin-bottom: 10px;
+        @media @mobile {
+         width: 100%;
+         margin-top: 40px;
+         font-size: 2.2rem;
+        }
+        
       }
 
       p {
         margin-bottom: 26px;
+        
       }
 
       .btn {
+        .custom-btn(100px, 50px, @btn-bg-color, @sea-shell);
+      
+        @media @mobile {
+          .custom-btn(300px, 50px, @btn-bg-color, @sea-shell);
+          
+        }
+  
+      }
+      @media @mobile{
+        order: 0;
+      }
+      @media @mobile{
+        order: 1;
+        
       }
     }
 
     .walk-img {
       width: 46%;
       margin-left: 4%;
```

Comment:
This property is pushing the image slightly to the right, making it off-center. Was there a reason for this?

Reply:
The 'order' property was used to rearrange the classes named as "walk-text" and "btn".
It is surely not the best practice to use it though. I have now updated the 'order' inside their classes.
Here is a screenshot if I do not use 'order' :
<img width="326" alt="without order property" src="https://user-images.githubusercontent.com/43893115/51502783-7c53bb80-1d8c-11e9-9a1d-1678b3e3cae7.png">